mmcv/runner/hooks/hook.py

```diff
@@ -65,3 +69,24 @@ def is_last_epoch(self, runner):
 
     def is_last_iter(self, runner):
         return runner.iter + 1 == runner._max_iters
+
+    def get_trigger_stages(self):
+        trigger_stages = set()
```

Comment:
If you are a hobbiest of one-liner,

```python
trigger_stages = set(stage for stage in Hook.stages if is_method_overridden(stage, Hook, self))
```

Reply:
Well, this one-line code will be splited into three lines because of 79-width limit.
So I feel it won't become more clear.